pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -312,8 +312,58 @@ public BrokerResponse handleRequest(JsonNode request, @Nullable RequesterIdentit
     long routingEndTimeNs = System.nanoTime();
     _brokerMetrics.addPhaseTiming(rawTableName, BrokerQueryPhase.QUERY_ROUTING, routingEndTimeNs - routingStartTimeNs);
 
+    // Set timeout in the requests
+    long timeSpentMs = TimeUnit.NANOSECONDS.toMillis(routingEndTimeNs - compilationStartTimeNs);
+    // Get the per-query timeout from the broker request query options if exists
+    long queryTimeoutMs = QueryOptions.getTimeoutMs(brokerRequest.getQueryOptions());
+    // Remaining time in milliseconds for the server query execution
+    // Use the max of offline table remaining time and realtime table remaining time for hybrid use case
+    long remainingTimeMs = 0;
+    // NOTE: Use query-level timeout if exists, or use table-level timeout if exists, or use instance-level timeout
```

Comment:
Can we move this line to just below Line 315 since this is the major logic of this code change in `BaseBrokerRequestHandler`?

Reply:
Done